packages/engine/Source/Scene/Model/Model.js

```diff
@@ -2060,7 +2058,7 @@ function updateClamping(model) {
   }
 
   const globe = scene.globe;
-  const ellipsoid = globe.ellipsoid;
+  const ellipsoid = defaultValue(globe?.ellipsoid, Ellipsoid.WGS84);
```

Comment:
Again, question about ES6 style guidelines:

* we're using `?` now instead of `defined()`?
* but also, we're not using `??` instead of `defaultValue`?

Reply:
We've used optional chaining, yes. See my [comment above about `??`](https://github.com/CesiumGS/cesium/pull/11604#discussion_r1450935430). We'd likely want to do that all at once since it would involve _replacing_ `defaultValue`;